core/src/test/java/org/apache/iceberg/TableTestBase.java

```diff
@@ -162,6 +167,19 @@ public class TableTestBase {
 
   static final FileIO FILE_IO = new TestTables.LocalFileIO();
 
+  // Mapping of Avro codec name used by Iceberg to name used by Avro (and appearing in Avro metadata
+  // under the key, avro.codec).
+  // In tests, we use the Iceberg name to specify the codec, and we verify the codec used by reading
+  // the Avro metadata and checking for the Avro name in avro.codec.
+  static final Map<String, String> AVRO_CODEC_NAME_MAPPING =
+      ImmutableMap.<String, String>builder()
+          .put(Avro.Codec.UNCOMPRESSED.name(), DataFileConstants.NULL_CODEC)
+          .put(Avro.Codec.ZSTD.name(), DataFileConstants.ZSTANDARD_CODEC)
+          .put(Avro.Codec.GZIP.name(), DataFileConstants.DEFLATE_CODEC)
+          .build();
+
+  static final long SNAPSHOT_ID = 987134631982734L;
```

Comment:
How about a name that signals that this is an example value? Like `EXAMPLE_SNAPSHOT_ID`?

Reply:
Actually, I see some tests with `DUMMY_SNAPSHOT_ID`. Can you reuse that instead of making a new one? You may need to move it here.